app/BUILD.bazel

```diff
@@ -582,6 +582,7 @@ kt_android_library(
         "//model:question_java_proto_lite",
         "//model:topic_java_proto_lite",
         "//third_party:androidx_recyclerview_recyclerview",
+        "//utility/src/main/java/org/oppia/android/util/parser/html:html_parser",
```

Comment:
This is interesting--which listener requires this package? It was my understanding that nearly all of the classes in this library are interfaces, so I wouldn't expect any to depend on HtmlParser.

Reply:
It didn't needed it actually ViewBindingShim.kt needed it so now I have added the parser dependency in app/shim package bazel